web/css/index.css

```diff
@@ -492,6 +492,8 @@ header.active {
   box-sizing: border-box;
   text-align: center;
   width: 10rem;
+  height: 4.8rem;
```

Comment:
Height specification of the main link element is now necessary because the three links' contents are no longer the same height